bin/wiby.js

```diff
@@ -1,12 +1,28 @@
 #!/usr/bin/env node
 
+require('dotenv').config()
 const test = require('../lib/test')
 const result = require('../lib/result')
 
-if (process.argv[2] === 'test') {
-  test()
+const args = require('yargs')
+  .option('test', {
+    alias: 't',
+    describe: 'Test your dependents'
+  })
+  .option('result', {
+    alias: 'r',
+    describe: 'Get the result of your tests'
+  })
+  .option('package', {
+    alias: 'pkg',
+    describe: 'Specific dependent to find test results for'
+  })
+  .argv
```

Comment:
```suggestion
  .strict()
  .help()
  .parse()
```
the "strict" means it will show the help when an invalid command is supplied; help enables the help; and .parse() both would throw if you made a typo instead of silently being undefined.

Reply:
Fixed